docker/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN dotnet restore DfE.FindInformationAcademiesTrusts
 RUN dotnet build DfE.FindInformationAcademiesTrusts -c Release
 RUN dotnet publish DfE.FindInformationAcademiesTrusts -c Release -o /app --no-build
 
-COPY ./script/web-docker-entrypoint.sh /app/docker-entrypoint.sh
+COPY ./docker/web-docker-entrypoint.sh /app/docker-entrypoint.sh
```

Comment:
If `docker build` is run from the `docker` folder as suggested in the GitHub action comment, then we can change `./script/` to just `./`